files/ko/web/javascript/reference/global_objects/array/every/index.html

```diff
@@ -148,44 +194,111 @@ <h2 id="폴리필">폴리필</h2>
     }
     return true;
   };
-}</pre>
-
-<h2 id="명세">명세</h2>
-
-<table class="standard-table">
- <tbody>
-  <tr>
-   <th scope="col">명세</th>
-   <th scope="col">상태</th>
-   <th scope="col">설명</th>
-  </tr>
-  <tr>
-   <td>{{SpecName('ES5.1', '#sec-15.4.4.16', 'Array.prototype.every')}}</td>
-   <td>{{Spec2('ES5.1')}}</td>
-   <td>초기 정의. JavaScript 1.6에서 구현됨.</td>
-  </tr>
-  <tr>
-   <td>{{SpecName('ES6', '#sec-array.prototype.every', 'Array.prototype.every')}}</td>
-   <td>{{Spec2('ES6')}}</td>
-   <td> </td>
-  </tr>
-  <tr>
-   <td>{{SpecName('ESDraft', '#sec-array.prototype.every', 'Array.prototype.every')}}</td>
-   <td>{{Spec2('ESDraft')}}</td>
-   <td> </td>
-  </tr>
- </tbody>
-</table>
-
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p>{{Compat("javascript.builtins.Array.every")}}</p>
-
-<h2 id="같이_보기">같이 보기</h2>
+}
+</pre>
+
+<h2 id="Examples">예제</h2>
+
+<h3 id="Testing_size_of_all_array_elements">모든 배열 요소의 크기 테스트</h3>
+
+<p>다음 예는 배열의 모든 요소가 10보다 더 큰지 테스트합니다.</p>
+
+<pre class="brush: js">
+function isBigEnough(element, index, array) {
+  return element &gt;= 10;
+}
+[12, 5, 8, 130, 44].every(isBigEnough);   // false
+[12, 54, 18, 130, 44].every(isBigEnough); // true
+</pre>
+
+<h3 id="Using_arrow_functions">화살표 함수 사용</h3>
+
+<p>
+  <a href="/ko/docs/Web/JavaScript/Reference/Functions/Arrow_functions">화살표 함수</a>는 같은
+  테스트에 대해 더 짧은 구문을 제공합니다.
+</p>
+
+<pre class="brush: js">
+[12, 5, 8, 130, 44].every(elem =&gt; elem &gt;= 10); // false
+[12, 54, 18, 130, 44].every(elem =&gt; elem &gt;= 10); // true</pre
+>
+
+<h3 id="Affecting_Initial_Array_modifying_appending_and_deleting">
+  초기 배열에 영향주기(수정, 추가, 삭제)
+</h3>
+
+<p>
+  아래 예제는 배열이 수정될 때 <code>every</code> 메서드의 행위를 테스트합니다. The following
+  examples tests the behavior of the <code>every</code> method when the array is modified.
```

Comment:
```suggestion
  아래 예제는 배열이 수정될 때 <code>every</code> 메서드의 행위를 테스트합니다.
```

잘 번역됐으니 원문을 삭제해주세요 :)